js/index.js

```diff
@@ -1,7 +1,65 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Tim";
+console.log("The driver's name is " + hacker1)
+
+let hacker2 = "Agustina";
+console.log("The navigator's name is " + hacker2)
 
 // Iteration 2: Conditionals
 
 
-// Iteration 3: Loops
\ No newline at end of file
+if(hacker1.length > hacker2.length){
+    console.log("The driver has the longest name, it has " + hacker1.length  + " characters.")
+  } else if(hacker2.length > hacker1.length){
+    console.log("It seems that the navigator has the longest name, it has " + hacker2.length + " characters.")
+  } else {
+    console.log("Wow, you both have equally long names, " + hacker1.length + " characters")
+  };
+  
+
+// Iteration 3: Loops
+
+// 3.1
+
+let upperCaseDriverName = "";
+
+for (let i = 0; i < hacker1.length; i++) {
+  upperCaseDriverName = upperCaseDriverName + " " + hacker1[i].toUpperCase()
+}
+console.log(upperCaseDriverName.trim());
+
+// 3.2
+
+let reversedNavigatorName = "";
+for (let i = hacker2.length - 1; i >=0; i--){
+  reversedNavigatorName += hacker2[i];
+}
+console.log(reversedNavigatorName)
+
+// 3.3
+
+if (hacker1.localeCompare(hacker2) === 1){
```

Comment:
Some browsers can return -2 (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare)
The best way to write this condition is either: 
hacker1.localeCompare(hacker2) > 0
hacker1.localeCompare(hacker2) < 0
hacker1.localeCompare(hacker2) === 0